demo/demo/model/bart_base_vocab_constraint.py

```diff
@@ -0,0 +1,43 @@
+import torch
+from transformers import AutoModelForSeq2SeqLM, AutoTokenizer
+
+_MODEL_DIR = f'models/bart_base/'
+
+MODEL_NAME = 'bart.base.vocab.input'
+
+model = AutoModelForSeq2SeqLM.from_pretrained(_MODEL_DIR)
+tokenizer = AutoTokenizer.from_pretrained(_MODEL_DIR)
+
+model = model.cuda()
+model.eval()
+
+GENERATION_CONFIG = {
+    "num_beams": 6,
+    "no_repeat_ngram_size": 3,
+    "early_stopping": False,
+    "min_length": 10,
+    "max_length": 60,
+    "limit_vocab_to_input": True,
+    "do_sample": True,
+}
+
+
+def produce_summary(source_text: str) -> str:
```

Comment:
try to avoid code duplication. if the only difference is the config. can you just pass that config instead of copying the entire file.

Reply:
done.